data/json/mapgen/survivor_forest_camp.json

```diff
@@ -63,6 +63,7 @@
         { "furn": "f_cattails", "x": [ 10, 16 ], "y": [ 21, 23 ], "repeat": [ 3, 7 ] },
         { "furn": "f_boulder_large", "x": [ 10, 16 ], "y": [ 21, 23 ], "repeat": [ 1, 3 ] }
       ],
+      "sealed_item": { "=": { "items": { "item": "farming_seeds", "chance": 100 }, "furniture": "f_plant_harvest" } },
```

Comment:
You probably want to make this `f_plant_seedling` instead, since those plants will be more or less freshly planted.